components/src/core/Drawer/DrawerHeader/DrawerHeader.tsx

```diff
@@ -14,6 +14,7 @@ import clsx from 'clsx';
 import { styled, SxProps, Theme } from '@mui/material/styles';
 import Box from '@mui/material/Box';
 import { unstable_composeClasses as composeClasses } from '@mui/base';
+import type {} from '@mui/material/themeCssVarsAugmentation';
```

Comment:
This should probably be defined once in the root index file instead of in a specific component file